tests/LogicItLab/Salesforce/MapperBundle/WsdlValidatorTestCaseTest.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Tests\LogicItLab\Salesforce\MapperBundle;
+
+use LogicItLab\Salesforce\MapperBundle\WsdlValidatorTestCase;
+
+class WsdlValidatorTestCaseTest extends WsdlValidatorTestCase
+{
+    public function modelAndWsdlDataProvider(): array
+    {
+        return [
+            [   sprintf('%s/Stubs', dirname(__FILE__)),
+                sprintf('%s/Resources/test.full.wsdl.xml', dirname(__FILE__))
+            ]
+        ];
+    }
+}
```

Comment:
In the projects this should be something like
`public function modelAndWsdlDataProvider(): array
    {
        return [
            [   $modelDirPath,
                $devWsdl
            ],
            [   $modelDirPath,
                $stagingWsdl
            ],
             [   $modelDirPath,
                $prodWsdl
            ],
        ];
    }`